src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,60 @@
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo,
+  handleSelectTodo: (todo: Todo) => void,
+  selected: boolean,
+};
+
+export const TodoItem: React.FC<Props> = ({
+  todo,
+  handleSelectTodo,
+  selected,
+}) => {
+  const {
+    id,
+    title,
+    completed,
+  } = todo;
+
+  return (
+    <tr data-cy="todo" className="">
```

Comment:
className attribute is empty. Either remove it or apply appropriate class names 😊

Reply:
Fix Vadim's comment